packages/grpc-authentication/src/index.ts

```diff
@@ -91,9 +91,9 @@ export class GrpcAuthentication extends GrpcConnection {
    * }
    * ```
    */
-  static async withKeyInfo(key: KeyInfo, host = defaultHost, debug = false) {
+  static async withKeyInfo(key: KeyInfo, host = defaultHost, debug = false, date?: Date) {
```

Comment:
maybe it's time to move this to `withKeyInfo(key, options)`? that options object is used all over the place.

Reply:
alternatively, we could move `host` last...

Reply:
Yes I'd like to use an options arg... I was just sneaking this in to avoid any API changes. But if we're ok with some API breakage here, then let's go with the options arg...

Reply:
oh, that's why i was mentioning. now is the moment since the next release already contains them